sideboard/tests/test_configuration.py

```diff
@@ -74,7 +74,7 @@ def config_overrides_set(self, monkeypatch):
     @pytest.fixture
     def plugin_dirs(self):
         module_path = '/fake/sideboard/plugins/test-plugin/test_plugin'
-        root_path = os.path.join(os.getcwd(), 'plugins', 'test-plugin')
+        root_path = os.path.join(config['plugins_dir'], 'test-plugin')
```

Comment:
It's probably fine. `root_path` was not supposed to be a real directory. I think I was just trying to test that `test-plugin` and `test_plugin` both work as the plugin's directory name.